db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
+      t.string :label_name
+      t.text :label_value
+      t.bigint :category_tag_id
+      t.bigint :tag_id
```

Comment:
Gave this comment on the other pr as well.
* I like to use
```ruby
t.belongs_to :tag, :type => :bigint
```

It has no difference but its clearer.

* I think you can drop the "category_tag_id" if the tag is connected to its category, i assume you can get it from the tag (.classifaction/.category) and place a ```:through``` ?